async_fuse/Cargo.toml

```diff
@@ -13,6 +13,7 @@ categories = ["filesystem"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+aligned-bytes = "0.1.4"
```

Comment:
how about use git dependency?

Reply:
`aligned-bytes` is an independent public library.
If there is any change in the future, I prefer to bump the version instead of the commit hash.